rich/text.py

```diff
@@ -242,6 +242,28 @@ def from_markup(
         rendered_text.overflow = overflow
         return rendered_text
 
+    @classmethod
+    def from_ansi(
+        cls,
+        text: str,
+        *,
+        justify: Optional["JustifyMethod"] = None,
+        overflow: Optional["OverflowMethod"] = None,
```

Comment:
Could you copy the other arguments from the Text constructor, i.e. style, no_wrap, end, and tab_size (no need for spans).

Reply:
If the `style=` kwarg is provided, what should that do, exactly?  Add to the existing styling, or replace it?

Reply:
Set it in the returned Text instance, it defines the base style.